clients/src/test/resources/log4j.properties

```diff
@@ -19,3 +19,5 @@ log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=[%d] %p %m (%c:%L)%n
 
 log4j.logger.org.apache.kafka=ERROR
+# We are testing for a particular INFO log message in DefaultSslEngineFactory
+log4j.logger.org.apache.kafka.common.security.ssl.DefaultSslEngineFactory=INFO
```

Comment:
You should be able to set the log level in the test that relies on this instead of setting it here in the file used for all tests.

Reply:
This was my first idea, too. But I tried several ways to do that programmatically and failed. AFAIK, slf4j does not provide any means to change the log level as it only acts as facade for logging frameworks such as log4j. But, importing log4j into the test cases to update the log level under the hood is currently prohibited by the stylechecker (log4j is blacklisted). Thus, changing the properties file seemed to be the least intrusive way to handle that issue.
Do you have a hint how the log level could be changed anyway?

Reply:
Still couldn't find a way to change the log level programmatically without changing any of the (quite reasonable) project configurations such as the stylecheck blacklist. But please not that the updated properties file is used only during testing. And the log level is changed now only exactly for the common name logging wrapper, none of the other classes is affected anymore.